jsonToJdl.js

```diff
@@ -73,6 +73,10 @@ const createJdlFromJson = (fileName, res) => {
                 applicationError[i] = ["Server Port cannot be empty"];
         }
 
+        var appFramework = false;
+        if (applications[i].applicationFramework == "go") {
```

Comment:
define list of blueprints
`if (applications[i].applicationFramework !== undefined && blueprints.contains(applications[i].applicationFramework)) {`